iroha/src/lib.rs

```diff
@@ -118,6 +121,15 @@ where
 
         //iroha_logger::info!(?config, "Loaded configuration");
 
+        let listen_addr = config.torii_configuration.torii_p2p_addr.clone();
+        iroha_logger::info!("Starting network on {}", &listen_addr);
```

Comment:
I understand that you call it network because we have several actors that keep connections. But in this case I think it should be `Starting peer om {}` or our users will not understand. Network is an internal naming.